documentation/src/docs/controller-service.md

```diff
@@ -517,275 +494,253 @@ the lock on the resource and hence permission to process the task. The
 The scale requests processing is then performed asynchronously on the
-request event processor.   
+Request Event Processor.   
 
 ### Retention Infrastructure
 
 The retention policy defines how much data should be retained for a
-given stream. This can be defined as _time based_ or _size based_. To apply
-this policy, Controller periodically collects StreamCuts for the stream
-and opportunistically performs truncation on previously collected stream
-cuts if policy dictates it. Since this is a periodic background work
-that needs to be performed for all streams that have a retention policy
-defined, there is an imperative need to fairly distribute this workload
+given Stream. This can be defined as _time based_ or _size based_. To apply
+this policy, Controller periodically collects `StreamCut`(s) for the Stream
+and opportunistically performs truncation on previously collected `StreamCut`(s) if policy dictates it. Since this is a periodic background work that needs to be performed for all Streams that have a retention policy defined, there is an imperative need to fairly distribute this workload
 across all available Controller instances. To achieve this we rely on
-bucketing streams into predefined sets and distributing these sets
+bucketing Streams into predefined sets and distributing these sets
 across Controller instances. This is done by using Zookeeper to store
 this distribution. Each Controller instance, during bootstrap, attempts
-to acquire ownership of buckets. All streams under a bucket are
+to acquire ownership of buckets. All Streams under a bucket are
 monitored for retention opportunities by the owning Controller. At each
-period, Controller collects a new stream cut and adds it to a
-retention set for the said stream. Post this it looks for candidate
-StreamCuts stored in retention set which are eligible for truncation
+period, Controller collects a new `StreamCut` and adds it to a
+retention set for the said Stream. Post this it looks for candidate
+`StreamCut`(s) stored in retention set which are eligible for truncation
 based on the defined retention policy. For example, in time based
-retention, the latest StreamCut older than specified retention period
+retention, the latest `StreamCut` older than specified retention period
 is chosen as the truncation point.
 
 ## Transaction Manager
 
 Another important role played by Controller is that of Transaction
-manager. It is responsible for creation and committing and abortion of
-transactions. Since Controller is the central brain and agency in our
-cluster, and is the holder of truth about stream, the writers request
+manager. It is responsible for creation, committing and abortion of
+Transactions. Since Controller is the central brain and agency in Pravega cluster, and is the holder of truth about Stream, the writers request
```

Comment:
I don't understand `central brain and agency`. Suggested rephrase for the whole paragraph:

```
Another important role played by Controller is that of Transaction manager. It is responsible for begining and 
ending Transactions. Controller plays an active role in providing guarantees for Transactions from the time 
they are created till the time they are committed or aborted. The Controller tracks each Transaction for their 
specified timeouts, and automatically aborts the transaction if the timeout exceeds.
```

Reply:
Rephrased!